app/javascript/Redux/store.js

```diff
@@ -0,0 +1,10 @@
+import { configureStore } from "@reduxjs/toolkit";
```

Comment:
- [ ] Kindly ensure your app works well enough. You are importing a package that you did not really install. Kindly fix that.

  ![utit](https://user-images.githubusercontent.com/54848363/197773072-3623de5b-2bc3-4549-bdfc-50225571cd4b.jpg)